### ISSUE TITLE
[Bug]: Python complaining about backslashes inside f-string expression in dropped file dupe resolution widget

### ISSUE THREAD

**Issue by Technoguyfication** [TagStudioDev/TagStudio#288]
### Checklist

- [X] I am using an up-to-date version.
- [X] I have read the [documentation](https://github.com/TagStudioDev/TagStudio/blob/main/doc/index.md).
- [X] I have searched existing [issues](https://github.com/TagStudioDev/TagStudio/issues).

### TagStudio Version

e5a0e5aa9bad9ff275002549be41474b4e89c0ba

### Operating System & Version

Windows 11

### Description

```
Traceback (most recent call last):
  File "C:\Users\***\GitHub\TagStudio\TagStudio\tag_studio.py", line 9, in <module>
    from src.qt.ts_qt import QtDriver
  File "C:\Users\***\GitHub\TagStudio\TagStudio\src\qt\ts_qt.py", line 86, in <module>
    from src.qt.modals.drop_import import DropImport
  File "C:\Users\***\GitHub\TagStudio\TagStudio\src\qt\modals\drop_import.py", line 173
    )
    ^
SyntaxError: f-string expression part cannot include a backslash
```

### Expected Behavior

App should launch as expected

### Steps to Reproduce

- Clone e5a0e5aa9bad9ff275002549be41474b4e89c0ba
- Launch Python module

### Logs

See description

**Comment by CyanVoxel** [comment]
Are you running a version of Python lower than 3.12?

**Comment by Technoguyfication** [comment]
> Are you running a version of Python lower than 3.12?

Looks like I am, `3.11.2`
Did not realize this project required >=3.12, in my defense everything else still works on 3.11 ;)

**Comment by CyanVoxel** [comment]
> Looks like I am, `3.11.2` Did not realize this project required >=3.12, in my defense everything else still works on 3.11 ;)

As far as we know everything else works on 3.11 and possibly 3.10, but haven't officially verified support for those versions again yet. With that being said I'd like to keep up that backwards compatibility if possible, so I'll continue along with the bug report and PR :)

**Comment by CyanVoxel** [comment]
Thanks for the fix! (Fixed in #289)
